cosmos/providers/dbt/parser/project.py

```diff
@@ -164,22 +215,38 @@ def _handle_config_file(self, path: Path) -> None:
         if not config_dict.get("models"):
             return
 
-        for config in config_dict["models"]:
-            model_name = config.get("name")
+        for model in config_dict["models"]:
+            model_name = model.get("name")
 
             # if the model doesn't exist, we can't do anything
             if model_name not in self.models:
                 continue
 
-            # parse out the config fields we can recognize
-
-            # 'tags' is either a string or list of strings
-            tags = config.get("tags", [])
-            if isinstance(tags, str):
-                tags = [tags]
+            # config_selectors
+            config_selectors = []
+            for selector in ["materialized", "schema", "tags"]:
```

Comment:
can we move the definition of the list of supported selectors to the top of the file as a constant? looks like we use it in multiple places